specification/apimanagement/resource-manager/readme.go.md

```diff
@@ -35,7 +35,7 @@ These settings apply only when `--tag=package-2018-01 --go` is specified on the
 Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
 
 ``` yaml $(tag) == 'package-2018-01' && $(go)
-output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2018-01-01/$(namespace)
+output-folder: $(go-sdk-folder)/services/stable/$(namespace)/mgmt/2018-01-01/$(namespace)
```

Comment:
Removing `preview` is sufficient as all packages not under `preview` are considered stable.